### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by bjornrl**
Walkthrough of how to set up Strapi with Homebrew and PostgreSQL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
 # Strapi application
 
-A quick description of your strapi application
+Setting up Strapi
+
+The first thing you do after forking the ACC-repo is to run "npm run build" and "npm run develop" to start the server, but as you will see, its not that easy.
+
+To get started with Strapi you need to install PostgreSQL, and for that you need Homebrew. HomeBrew is a package manager that takes care of everything NPM or Yarn doesn´t. Read more about it on https://brew.sh/
```
Comment (oriooctopus) [inline README.md:7]: Unfortunately this is specific to mac, for this part can you make a separate section called "Setting up for mac" and I'll ask essam to create a "Setting up for windows" section

**Review by oriooctopus — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
 # Strapi application
 
-A quick description of your strapi application
+Setting up Strapi
+
+The first thing you do after forking the ACC-repo is to run "npm run build" and "npm run develop" to start the server, but as you will see, its not that easy.
```
Comment (oriooctopus) [inline README.md:5]: also npm install. Can you make this more of a bullet point list or a step by step?

File: README.md (hunk 1 above)
Comment (oriooctopus) [reply README.md:7]: Can you make a "Setting up for mac section", right now this implies it works for both operating systems

File: README.md (hunk 2 above)
Comment (oriooctopus) [inline README.md:5]: use syntax highlighting on code commands

File: README.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+
+To get started with Strapi you need to install PostgreSQL, and for that you need Homebrew. HomeBrew is a package manager that takes care of everything NPM or Yarn doesn´t. Read more about it on https://brew.sh/
+
+This is the website that will get you started. Most of the steps here are taken directly from there. https://www.moncefbelyamani.com/how-to-install-postgresql-on-a-mac-with-homebrew-and-lunchy/ PS: This is only for mac. Not sure if it works the same way with Linux or Windows.
+
+To install HomeBrew, run this command in your terminal:
```
Comment (oriooctopus) [inline README.md:11]: Stuff like this shoudl have a header called "Installing homebrew"

**Review by oriooctopus — CHANGES_REQUESTED** [review]
Can you use https://markdownlivepreview.com/ to write this in markdown? Look at the readmes of other repos https://github.com/strapi/strapi/ for example look at their use of headings, bullet lists, etc. That is standard for any repo.
Also can you add a section on github development workflow? (Forking, Pulling from upstream, pushing to origin, include that they need to add upstream because it's not there by default). There are multiple levels of headers, you'll want a big header for "Github Development Workflow" and smaller headers for its subsections. Same for the above

File: README.md [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
+###### This section only works for mac and linux
 
-To get started with Strapi you need to install PostgreSQL, and for that you need Homebrew. HomeBrew is a package manager that takes care of everything NPM or Yarn doesn´t. Read more about it on https://brew.sh/
+Windows sections will be added later. For now check out these links to set up:
 
-This is the website that will get you started. Most of the steps here are taken directly from there. https://www.moncefbelyamani.com/how-to-install-postgresql-on-a-mac-with-homebrew-and-lunchy/ PS: This is only for mac. Not sure if it works the same way with Linux or Windows.
+- Windows: https://www.microfocus.com/documentation/idol/IDOL_12_0/MediaServer/Guides/html/English/Content/Getting_Started/Configure/_TRN_Set_up_PostgreSQL.htm
 
-To install HomeBrew, run this command in your terminal:
+#### Proceedings for Mac users
```
Comment (oriooctopus) [inline README.md:11]: This is to general. Instead of "Proceedings" say something like "Setting up database"

File: README.md [hunk 5]
```diff
@@ -1,16 +1,35 @@
...
 
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+1. Install Homebrew by running command: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`. _(This will ask for you password and confirmation of download)_
+1. Check that homebrew is up to date and healty by running: `brew update` then `brew doctor`
```
Comment (oriooctopus) [inline README.md:14]: Why are there a bunch of "1."s?

File: README.md [hunk 6]
```diff
@@ -1,16 +1,35 @@
...
+1. Install Postgres by running: `brew install postgresql`
+1. Start Postgres service by running: `brew services start postgresql`
+1. If you get error: "Error: role 'postgres' does not exist", run command: `/usr/local/opt/postgres/bin/createuser -s postgres`
+
+_Remember to run `npm run build` and `npm run develop` after doing these steps._
```
Comment (oriooctopus) [inline README.md:19]: This isn't related to setting up the database, it should be in a different section

File: README.md [hunk 7]
```diff
@@ -1,16 +1,35 @@
...
+
+_PS: The step “Downloading Command Line Tools for Xcode” can take very long._
+My best suggestion for that is checking out this discussion: https://github.com/Homebrew/discussions/discussions/131
+
+And especially this comment:
+
+Apple's XCode command-line installer is...quite uncommunicative. You have at least two choices: 1. Wait for as long as it takes...or the installer errors out. 2. Go to https://developer.apple.com/download/more/ (you'll need to sign up for an Apple developer account if you don't have one), then download and install the latest Command Line Tools package that your OS supports.
```
Comment (oriooctopus) [inline README.md:26]: Use a blockquote here

**Review by oriooctopus — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (bjornrl) [reply README.md:14]: On the markdown tutorial, it says to use 1.s to get a numbered list

File: README.md (hunk 6 above)
Comment (oriooctopus) [reply README.md:19]: @bjornrl

**Review by oriooctopus — APPROVED** [review]

**Pull request merged**